lib/Remotery.c

```diff
@@ -3665,9 +3675,12 @@ static rmtError ThreadSampler_Constructor(ThreadSampler* thread_sampler)
     thread_sampler->next = NULL;
 
     // Set the initial name to Thread0 etc.
-    thread_sampler->name[0] = 0;
-    strncat_s(thread_sampler->name, sizeof(thread_sampler->name), "Thread", 6);
-    itoahex_s(thread_sampler->name + 6, sizeof(thread_sampler->name) - 6, AtomicAdd(&countThreads, 1));
+    GetDebuggerThreadName(thread_sampler->name);
```

Comment:
What's the reason for doing this? Do the initial `PR_GET_NAME` names returned make more sense than `Thread0` etc. on POSIX?